### ISSUE TITLE
pagination feature

### ISSUE THREAD

**Issue by aga2442** [Automattic/WP-Job-Manager#2685]
when the pagination feature is used at the bottom of the page with job listings in any theme, no change occurs in the url field. we get a lot of negative feedback from visitors on this issue.

for example: the visitor scrolls to the 5th page while reviewing the job listings, reviews any listings and when he presses the back button on the page, he returns to the first row of all listings. this situation has a negative effect on many visitors.

https://wordpress.org/support/topic/pagination-not-working-in-job-listing-archive/

https://wordpress.org/support/topic/adding-configuring-pagination-on-single-company-php/

you have a few support messages on these pages in the past, but in my opinion, it would be very useful to fix this problem with a permanent solution.

I have also created a support message about this issue.

https://wordpress.org/support/topic/we-are-waiting-for-a-permanent-solution-to-2-important-problems-as-all-site-owne/

They suggested that I also submit a request on github.

**Comment by jinnypark** [comment]
7528864-zen

**Comment by Dumorya** [comment]
I totally agree with @aga2442 , you should at least propose a simple non AJAX pagination with URL change.

**Comment by igotdes** [comment]
Also an issue reported in this forum thread: https://wordpress.org/support/topic/pagination-issues-21/

**Comment by muellerelias** [comment]
Hey, is there any update on this?
Or can i make a small javascript, snipped, whichtake the page query parameter and trigger the pagenation?

**Comment by muellerelias** [comment]
Hey, is there any update?
Since Google also prefers real links via a-tag rather than Ajax.
